bzl/workspace.bzl

```diff
@@ -170,4 +170,28 @@ def plaidml_workspace():
         build_file = clean_dep("//bzl:zlib.BUILD"),
     )
 
+    http_archive(
+        name = "opencl_headers",
+        url = "https://github.com/KhronosGroup/OpenCL-Headers/archive/v2020.06.16.zip",
+        sha256 = "518703d3c3a6333bcf8e4f80758e4e98f7af30fbd72a09fe8c2673da1628d80c",
+        strip_prefix = "OpenCL-Headers-2020.06.16",
+        build_file = clean_dep("//vendor/opencl_headers:opencl_headers.BUILD"),
+    )
+
+    http_archive(
+        name = "opencl_hpp_headers",
+        url = "https://github.com/KhronosGroup/OpenCL-CLHPP/archive/v2.0.12.zip",
+        sha256 = "127936b3a5ef147f23b85fb043599d1480e9e57acabe2d2a67c5dac05aa4ad70",
+        strip_prefix = "OpenCL-CLHPP-2.0.12",
+        build_file = clean_dep("//vendor/opencl_hpp_headers:opencl_hpp_headers.BUILD"),
+    )
+
+    http_archive(
```

Comment:
FWIW -- we used to do this, since it got us an ICD loader with the most recent OpenCL API (so we could link, and then we could dynamically probe devices to see what they actually supported).  IIRC there was some problem with this approach, though, so we switched to using the system's OpenCL ICD, and using a shim layer that dynamically loaded the system OpenCL ICD, forwarding supported calls.